libs/execution_context/ebpf_maps.c

```diff
@@ -733,7 +754,7 @@ _update_hash_map_entry(
     if ((entry_count >= map->ebpf_map_definition.max_entries) &&
         (ebpf_hash_table_find((ebpf_hash_table_t*)map->data, key, &value) != EBPF_SUCCESS) &&
         !_reap_oldest_map_entry(map))
-        result = EBPF_INVALID_ARGUMENT;
+        result = EBPF_OUT_OF_SPACE;
```

Comment:
Why not use `EBPF_INSUFFICIENT_BUFFER`

Reply:
EBPF_INSUFFICIENT_BUFFER would map to ERROR_INSUFFICIENT_BUFFER

```122 (0x7A) The data area passed to a system call is too small.```

Which suggests the user needs to pass a bigger buffer. In this case, the condition is that the map is full.